docker/dev_env/run.sh

```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env bash
+
+set -e
+
+host_docker_gid="$(getent group docker | cut -d: -f3)"
+
+suppress_output() {
+  "$@" 2>/dev/null 1>/dev/null
+}
+
+opener_home="/home/opener"
+
+run_args=(
+  -i
+  --rm
+  --name openverse-dev-env
+  --env "OPENVERSE_PROJECT=$OPENVERSE_PROJECT"
+  --env "TERM=xterm-256color"
+  --user "$UID:$host_docker_gid"
+  --network host
+  # Bind the repo to the same exact location inside the container so that pre-commit
+  # and others don't get confused about where files are supposed to be
+  -v "$OPENVERSE_PROJECT:$OPENVERSE_PROJECT:rw,z"
+  --workdir "$OPENVERSE_PROJECT"
+  # Save the home directory of the container so we can reuse it each time
+  --mount "type=volume,src=openverse-dev-env,target=$opener_home"
+  # Expose the host's docker socket to the container so the container can run docker/compose etc
+  -v /var/run/docker.sock:/var/run/docker.sock
+)
+
+if [ -t 0 ]; then
+  run_args+=(-t)
+fi
+
+host_pnpm_store="$(pnpm store path 2>/dev/null || echo)"
+
+# Share the pnpm cache with the container, if it's available locally
+if [ "$host_pnpm_store" != "" ]; then
+  pnpm_home="$(dirname "$host_pnpm_store")"
+  run_args+=(
+    --env PNPM_HOME="$pnpm_home"
+    -v "$pnpm_home:$pnpm_home:rw,z"
+  )
+fi
+
+# Share the PDM cache with the container, if it's available locally
+# --quiet so PDM doesn't repeatedly fill the console with update messages
+# if they're enabled
+if [ "$(pdm --quiet config install.cache)" == "True" ]; then
```

Comment:
This comment and the use of `--quiet` in both PDM evocations are meant to address those PDM update messages.

Can you play around and see why `--quiet` isn't doing that for you? It seemed to work for me, but maybe I missed some edge case.

Reply:
Also, for what it's worth, you should upgrade PDM :grin:

Reply:
@sarayourfriend It looks like the output is not suppressed even when I run the command outside `ov` in my own shell 🤔 

```bash
$ pdm --quiet config install.cache
False
INFO: PDM 2.15.1 is installed, while 2.15.3 is available.
Please run `pipx upgrade pdm` to upgrade.
Run `pdm config check_update false` to disable the check.
```

Reply:
Ah! I'd run into this before with the `-v` flag for debugging errors. Thanks for sorting it out.